### ISSUE TITLE
Geant4 build failure on ubuntu-22.04

### ISSUE THREAD

**Issue by MilanKlausz** [mctools/dgcode#25]
Geant4 (10.04.p03) build [fails](https://github.com/mctools/dgcode_actions/actions/runs/3522329770) on all ubuntu-22.04 setups in the GitHub workflow runs. 
Update to Geant4 11.x [#2](https://github.com/mctools/dgcode/issues/2) might solve this issue, so we are not planning to directly deal with it. 
Until then, in order to avoid repeatedly trying to install Geant4 and failing after ~30 min, Geant4 is not installed in the workflow runs using ubuntu-22.04.

Once the update to Geant4 11.x is ready, the special handling of ubuntu-22.04 in the [run_tests.yml](https://github.com/mctools/dgcode_actions/blob/main/.github/workflows/run_tests.yml) workflow should be removed.
From:
``` 
INSTALL_GEANT4: ${{ inputs.without-geant4 != true && (matrix.os !='ubuntu-22.04') }} #temp solution until Geant4 update
```
To
``` 
INSTALL_GEANT4: ${{ inputs.without-geant4 != true }}
```

**Comment by MilanKlausz** [comment]
My suggested recipe for testing is:
- creating a new branch in dgcode, where the default geant4 version in the dgdepfixer set to the 11.x
- creating a new branch in dgcode_actions, where the special handling of ubuntu-22.04 is eliminated (as written above), to allow Geant4 installation.
- Manually triggering the run_tests.yml workflow in the dgcode_actions repository's [actions tab](https://github.com/mctools/dgcode_actions/actions/workflows/run_tests.yml), with the following settings: 
    - Selecting the dgcode_actions branch from the `Use workflow from` dropdown list
    - Selecting the framework branch by typing its name to the `Framework branch to use` field
    - Selecting a single setup from the `Software setup(s) to use` dropdown list with _ubuntu-22.04_ 
       OR selecting '_all_', and typing _ubuntu-22.04_ in either of the `Setup filter - include if (...)` fields (to run all setups with ubuntu-22.04).

**Comment by tkittel** [comment]
I have to think about this a bit. It might indeed be that this will need a new G4 version, since I am not able to spend a week to find a workaround in the short term. A workaround would either take the form of a particular compilation flag, or to find a patch (for our patched) G4 10.4...
